alphagov/paas-cf
# [#156091458] Merge 000 and 010 CF manifests

## What

Merge the 000 and 010 CF manifests. This brings us a little bit closer to [cf-deployment.yml](https://github.com/cloudfoundry/cf-deployment/blob/master/cf-deployment.yml).

Adjust some whitespace to look a bit more like cf-deployment. Re-order some keys etc.

Remove a test that started failing. It asserted that the order matched upstream order. We'll soon be using upstream directly, so this won't be necessary.

## How to review

- Deploy master to dev env
- Deploy this to dev env
- Ensure bosh shows no changes

## Who can review

Anyone but @camelpunch
